src/SmartComponents/BaselinesPage/EditBaseline/EditBaselineKebab/EditBaselineKebab.js

```diff
@@ -0,0 +1,99 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Dropdown, KebabToggle, DropdownItem } from '@patternfly/react-core';
+
+import DeleteFactModal from '../DeleteFactModal/DeleteFactModal';
+
+class EditBaselineKebab extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            kebabOpened: false,
+            modalOpened: false
+        };
+
+        this.toggleKebab = () => {
+            const { kebabOpened } = this.state;
+            this.setState({
+                kebabOpened: !kebabOpened
+            });
+        };
+
+        this.toggleModalOpened = () => {
+            const { modalOpened } = this.state;
+            this.setState({
+                modalOpened: !modalOpened,
+                kebabOpened: false
+            });
+        };
+    }
+
+    isFactSelected = () => {
+        const { editBaselineTableData } = this.props;
+        let isDisabled = true;
+
+        editBaselineTableData.forEach((fact) => {
+            if (fact.selected) {
+                isDisabled = false;
+            }
+
+            if (Array.isArray(fact[2]) && fact[2].length > 0) {
```

Comment:
I'm starting to wonder if we should introduce some constants to make things more eadable across all these components that deal with arrays. Either:

```
BASELINE_SUB_FACTS = 2

if (Array.isArray(fact[BASELINE_SUB_FACTS]) && fact[BASELINE_SUB_FACTS].length > 0) {
```

or with a function:

```
isCategory(fact) {
  let subfacts = fact[2];
  if (Array.isArray(subfacts) && subfacts.length > 0) {
    return true
  } else 
   return false;
  }
}

baseline_sub_facts(fact) {
    return fact[2];
}

if (isCategory(fact)) {
  baseline_sub_facts(fact).forEach(subFact) => {
```